src/elements/core/styles/core.scss

```
@@ -154,6 +154,52 @@ input[data-sbb-time-input] {
   }
 }
 
+// Target the slotted `sbb-image` which are generally wrapped by a <figure> (therefore are not reachable with the :slotted)
+// Apply the brightness effect on mouse hover
+// TODO Move back to the teaser components when the global css refactoring happens
+:is(sbb-teaser, sbb-teaser-hero, sbb-teaser-product) {
+  & [slot='image']:is(sbb-image, img),
+  & [slot='image'] :is(sbb-image, img) {
+    will-change: filter;
+    filter: brightness(var(--sbb-teaser-image-brightness, 1));
+    transition: filter var(--sbb-teaser-image-animation-duration)
+      var(--sbb-teaser-image-animation-easing);
+  }
+}
+
+// TODO Move back to the teaser components when the global css refactoring happens
+:is(sbb-teaser-product, sbb-teaser-product-static) {
+  // Reset sbb-image border radius in order to control it from teaser product.
+  sbb-image {
+    --sbb-image-border-radius: 0;
+  }
+
+  img {
+    place-self: stretch;
+    object-fit: cover;
+    aspect-ratio: 16 / 9;
+  }
+}
+
+// TODO Move back to the teaser components when the global css refactoring happens
+:is(sbb-teaser) {
+  & [slot='image']:is(sbb-image, img),
+  & [slot='image'] :is(sbb-image, img) {
+    transition-property: filter, transform;
+    will-change: filter, transform;
+    transform: scale(var(--sbb-teaser-scale, 1));
```

We can now use it :-)
```suggestion
    scale: var(--sbb-teaser-scale, 1);
```